### ISSUE TITLE
Typeahead does not work using VoiceOver

### ISSUE THREAD

**Issue by amyeverett1** [UKHomeOffice/typeahead-aria#4]
Please see attached video in ZIP file. 

The typeahead does not correctly read out country options available to the user. Seems to read part of the country name or sometimes just a random selection of letters from another country in the list.

[typeahead.mp4.zip](https://github.com/UKHomeOffice/typeahead-aria/files/580649/typeahead.mp4.zip)

**Comment by colinrotherham** [comment]
Hi @amyeverett1, do you have `highlight: true` set in your options?

That adds markup that styles fragments of the results names (to highlight the bits that match your search keywords). Turning that off could help?

I've opened a pull request here to merge in a lot of bug fixes from another fork:
https://github.com/UKHomeOffice/typeahead-aria/pull/8

That could help too.

**Comment by amyeverett1** [comment]
Thanks @colinrotherham - I'll ask the devs if they can take a look and see if your bug fixes / adding in highlight:true fixes the issue. :)

**Comment by colinrotherham** [comment]
@amyeverett1 No problem. Try without `highlight: true` or `highlight: false` though, I think it's the highlighting that's probably fragmenting what VoiceOver reads.
